app/features/modelling/line-plot/index.js

```diff
@@ -1,5 +1,37 @@
 import { Chart } from 'chart.js/auto';
 
+function getXYData(data) {
+  return {
+    data: data.x.map((x, index) => ({ x, y: data.y[index] })),
+    pointStyle: 'circle',
+    backgroundColor: '#ffffff',
+    borderColor: '#000000'
+  }
+}
```

Comment:
This is new. Otherwise, the rest of the code in this PR should be unchanged from the original.